vm/src/stdlib/winapi.rs

```diff
@@ -402,4 +408,58 @@ mod _winapi {
         })
         .map(drop)
     }
+
+    use std::ffi::{OsStr, OsString};
+    use std::os::windows::prelude::*;
+    pub trait ToWide {
+        fn to_wide(&self) -> Vec<u16>;
+        fn to_wide_null(&self) -> Vec<u16>;
+    }
+    impl<T> ToWide for T where T: AsRef<OsStr> {
+        fn to_wide(&self) -> Vec<u16> {
+            self.as_ref().encode_wide().collect()
+        }
+        fn to_wide_null(&self) -> Vec<u16> {
+            self.as_ref().encode_wide().chain(Some(0)).collect()
+        }
+    }
+    pub trait FromWide where Self: Sized {
+        fn from_wide_null(wide: &[u16]) -> Self;
+    }
+    impl FromWide for OsString {
+        fn from_wide_null(wide: &[u16]) -> OsString {
+            let len = wide.iter().take_while(|&&c| c != 0).count();
+            OsString::from_wide(&wide[..len])
+        }
+    }
+
+    const MAX_PATH: usize = 260;
```

Comment:
@youknowone Do you have any idea to set MAX_PATH?

Reply:
I am not an expert of windows API, but reusing it from windows-rs?

https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/Foundation/constant.MAX_PATH.html

I found another article about it: https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry

Reply:
https://github.com/python/cpython/blob/v3.10.5/Modules/_winapi.c#L1309-L1324
I found that in the implementation of cpython, it uses ``WCHAR[MAX_PATH]``(``WCHAR = ctypes.c_wchar``
) and ``ctypes.c_wchar`` has a fixed size of 2 bytes https://github.com/nvaccess/nvda/issues/8981.
So it's fine to reuse ``MAX_PATH`` of windows-rs.